src/components/ShareMyLocationButton.js

```diff
@@ -19,6 +19,7 @@ const ShareMyLocationButton = () => {
     const lng = position.coords.longitude;
     const newMapCenter = { lat, lng };
     setUserCenterMap(newMapCenter);
+    setNewCenterMap({ ...newMapCenter, isDragged: false });
```

Comment:
I'm not sure but I think we can't drag the map 👀 (re-center map).

Reply:
**Optional suggestion:** I think one possible solution for this ^^^. It's adding a control: if we dragged the map & share your location, the map center would be the location where you move the map (`center = dragLocation`). Another hand, if you didn't drag the map & share your location, the map center would be your location or user location (`center = userLocation`). Finally `setNewCenterMap(center)`.

Reply:
@alxmcr, if you run the app locally, you could see the map updates both 1) if the user moves; 2) if the user drags it manually